ui/logviewer/App.jsx

```diff
@@ -214,6 +225,30 @@ class App extends React.PureComponent {
     }
   };
 
+  logFormatter = (log) => {
+    if (/INFO - GECKO\(\d+\)/.test(log)) {
+      // Do nothing
+    } else if (/INFO - TEST-(OK)|(PASS)/.test(log))
+      return <span style={{ color: 'green' }}>{log}</span>;
+    else if (/INFO - TEST-UNEXPECTED-FAIL/.test(log))
+      return <span style={{ color: 'red' }}>{log}</span>;
```

Comment:
You don't need to highlight this because we already do that by default. That is what is being highlighted with the `highlight` prop.